dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/SubmissionFormRestRepository.java

```diff
@@ -65,4 +85,13 @@ public SubmissionFormRest findOne(Context context, String submitName) {
     public Class<SubmissionFormRest> getDomainClass() {
         return SubmissionFormRest.class;
     }
+
+    public void reload() throws DCInputsReaderException {
```

Comment:
Can we add JavaDocs here?  It looks like (to me) this method should say something like:  `Reload the current Submission Form configuration based on the currently supported locales. This method can be used to force a reload if the configured supported locales change.`